### ISSUE TITLE
Criar APP Suppliers

### ISSUE THREAD

**Issue by LucianocLopes** [LucianocLopes/DRP01-PJI240-SALA-006GRUPO-006#2]
a ideia neste app é conter o modelo de banco de dados principal, Fornecedores, ter as responsavilidade do CRUD.

**Comment by David-Souza** [comment]
Seguiremos com a pagina início sendo a exibição da tabela? Eu adoraria mexer no CRUD.

**Comment by LucianocLopes** [comment]
Eu havia pensado na página inicial um Dashboard ou algo mais simples mais que como facilidade uma representação gráfica básica de dados para demonstrar alguns pontos importantes.

**Comment by David-Souza** [comment]
Então o input de dados no banco de dados será só via a API? Não haverá mais o formulário anterior?

**Comment by LucianocLopes** [comment]
Não, o app API gera urls onde são possíveis operações com todos os verbos, fazendo com que qualquer aplicação que contenha essas urls, tenham a possibilidade de realizar tais operações. Já internamente o projeto está desenhando para uma página inicial com um Dashboard,  ao clicar no link Fornecedores levará para uma tabela com todos os fornecedores já cadastrados, com uma paginação de 20 fornecedores por vez e as opções de CRUD a partir de botões e links que estão na tabela. Dentro do detalhe do cadastro de fornecedores poderá ter informações mais específicas do fornecedor e cadastros de outras possíveis informações, tais como Telefones, Contratos e etc.
